crates/turbopack-mdx/src/lib.rs

```diff
@@ -31,12 +34,51 @@ fn modifier() -> StringVc {
     StringVc::cell("mdx".to_string())
 }
 
+/// Subset of mdxjs::Options to allow to inherit turbopack's jsx-related configs
+/// into mdxjs.
+#[turbo_tasks::value(shared)]
+#[derive(PartialOrd, Ord, Hash, Debug, Clone)]
+pub struct MdxTransformOptions {
+    pub development: bool,
+    pub preserve_jsx: bool,
+    pub jsx_runtime: Option<String>,
+    pub jsx_import_source: Option<String>,
+    pub filepath: Option<FileSystemPathVc>,
```

Comment:
```suggestion
```

Why would be pass the filepath when we already have this info from `source.ident().path()`